pkg/ceph/client/image_test.go

```diff
@@ -91,3 +91,103 @@ func TestCreateImage(t *testing.T) {
 	assert.True(t, createCalled)
 	createCalled = false
 }
+
+func TestListImageLogLevelInfo(t *testing.T) {
+	executor := &exectest.MockExecutor{}
+	context := &clusterd.Context{Executor: executor}
+
+	var images []CephBlockImage
+	var err error
+	listCalled := false
+	emptyListResult := false
+	executor.MockExecuteCommandWithOutput = func(debug bool, actionName string, command string, args ...string) (string, error) {
+		switch {
+		case command == "rbd" && args[0] == "ls" && args[1] == "-l":
+			listCalled = true
+			if emptyListResult {
+				return `[]`, nil
+			} else {
+				return `[{"image":"image1","size":1048576,"format":2},{"image":"image2","size":2048576,"format":2},{"image":"image3","size":3048576,"format":2}]`, nil
+
+			}
+		}
+		return "", fmt.Errorf("unexpected ceph command '%v'", args)
+	}
+
+	images, err = ListImages(context, "foocluster", "pool1")
+	assert.Nil(t, err)
+	assert.NotNil(t, images)
+	assert.True(t, len(images) == 3)
+	assert.True(t, listCalled)
+	listCalled = false
+
+	emptyListResult = true
+	images, err = ListImages(context, "foocluster", "pool1")
+	assert.Nil(t, err)
+	assert.NotNil(t, images)
+	assert.True(t, len(images) == 0)
+	assert.True(t, listCalled)
+	listCalled = false
+}
+
+func TestListImageLogLevelDebug(t *testing.T) {
```

Comment:
Looks like this unit test failed in the Jenkins build: https://jenkins.rook.io/blue/organizations/jenkins/rook%2Frook/detail/PR-947/2/pipeline

```
--- FAIL: TestListImageLogLevelDebug (0.00s)
	assertions.go:237: 
                        
	Error Trace:	image_test.go:180
		
	Error:		Expected nil, but got: &errors.errorString{s:"unmarshal failed: invalid character '\\\\' after top-level value.  raw buffer response: []\\n[{\"image\":\"image1\",\"size\":1048576,\"format\":2},{\"image\":\"image2\",\"size\":2048576,\"format\":2},{\"image\":\"image3\",\"size\":3048576,\"format\":2}]"}
		
	assertions.go:237: 
                        
	Error Trace:	image_test.go:181
		
	Error:		Expected value not to be nil.
		
	assertions.go:237: 
                        
	Error Trace:	image_test.go:182
		
	Error:		Should be true
```
Are you seeing the same failure locally?

Reply:
Sorry but on my windows laptop I'm currently not able to run the tests...but I will try bring this test case somehow to work

Reply:
The tests seem to run, a carriage return messed up the result